web-server/pages/api/resources/teams/[team_id]/bookmarked_repos.ts

```diff
@@ -0,0 +1,40 @@
+import * as yup from 'yup';
+
+import { getTeamRepos } from '@/api/resources/team_repos';
+import { Endpoint, nullSchema } from '@/api-helpers/global';
+import { Table } from '@/constants/db';
+import { uuid } from '@/utils/datatype';
+import { db } from '@/utils/db';
+
+const pathSchema = yup.object().shape({
+  team_id: yup.string().uuid().required()
+});
+
+const endpoint = new Endpoint(pathSchema);
+
+endpoint.handle.GET(nullSchema, async (req, res) => {
+  if (req.meta?.features?.use_mock_data) {
+    return res.send([1, 2, 3].map(() => uuid()));
```

Comment:
`[1,2,3]` seems very odd, maybe directly use` [uuid(),uuid(),uuid()]` or declare it inside a mock file